kazoo/protocol/connection.py

```diff
@@ -380,7 +381,8 @@ def _read_response(self, header, buffer, offset):
             # Determine if watchers should be registered
             watcher = getattr(request, 'watcher', None)
             if not client._stopped.is_set() and watcher:
-                if isinstance(request, GetChildren):
+                if (isinstance(request, GetChildren)
```

Comment:
@saffroy do you know if `GetChildren` is even used? Or is it always `GetChildren2`?

Reply:
Ah, that makes sense. There's a straightforward way to fix this linting issue, I'll leave a comment above